opentracing-api/src/main/java/io/opentracing/SpanContext.java

```diff
@@ -26,6 +26,24 @@
  * @see Span#getBaggageItem(String)
  */
 public interface SpanContext {
+    /**
+     * Return the ID of the trace.
+     *
+     * Globally unique. Every span in a trace shares this ID.
+     *
+     * @return the trace ID for this context.
```

Comment:
what about returning empty string and never null?

Reply:
Oh, good point.